src/features/common/Layout/Navbar/index.tsx

```diff
@@ -120,54 +127,111 @@ export default function NavbarComponent(props: any) {
           if (SingleLink) {
             if (link === 'me' && SingleLink.visible) {
               return (
-                <button id={'navbarActiveIcon'} key={link} onClick={() => gotoUserPage()} className={'linkContainer'}>
+                <button
+                  id={'navbarActiveIcon'}
+                  key={link}
+                  onClick={() => gotoUserPage()}
+                  className={'linkContainer'}
+                >
                   <div className={'link_icon'}>
                     <UserProfileIcon />
                   </div>
-                  <p className={router.pathname === SingleLink.onclick ? 'active_icon' : ''}>
-                    {isAuthenticated && userInfo && SingleLink.loggedInTitle ? t('common:' + SingleLink.loggedInTitle) : t('common:' + SingleLink.title)}
+                  <p
+                    className={
+                      router.pathname === SingleLink.onclick
+                        ? 'active_icon'
+                        : ''
+                    }
+                  >
+                    {isAuthenticated && userInfo && SingleLink.loggedInTitle
+                      ? t('common:' + SingleLink.loggedInTitle)
+                      : t('common:' + SingleLink.title)}
                   </p>
                 </button>
-              )
+              );
             }
-            if (link === 'about' && SingleLink.visible) {
+            if (
+              link === 'about' &&
+              SingleLink.visible 
+            ) {
               SingleLink = {
                 ...SingleLink,
-                onclick: `${SingleLink.onclick}${i18n.language === 'de' && (process.env.TENANT === 'planet' || process.env.TENANT === 'ttc') ? 'de' : ''}`
-              }
+                onclick: `${SingleLink.onclick}${
+                  process.env.TENANT === 'planet' ||
+                  process.env.TENANT === 'ttc' ? lang_path[i18n.language] : ""}`
```

Comment:
In this line, you also have to check if `lang_path[i18n.language]` exists or this will result in a 404 error. 
https://a.plant-for-the-planet.org/undefined